### DISCUSSION TITLE
How to get older distros?

### DISCUSSION THREAD

**Discussion by Jefferson111** [microsoft/WSL#6501] [General]
**Is your feature request related to a problem? Please describe.**
Hi, I wonder if its possible to install older wsl distros?

I would like to install 18.04.04 LTS Ubuntu, but Microsoft store only provides 18.04.05 LTS Ubuntu

I would like to run some software that is supported in 18.04.04 LTS Ubuntu.

**Describe the solution you'd like**
I would like to be able to choose the specific patch I can install for Ubuntu. 
E.g. 18.04.04 LTS Ubuntu

**Describe alternatives you've considered**
Don't use WSL which defeats the purpose of the existence of WSL

**Additional context**

**Comment by crramirez** [comment]
Hello @Jefferson111 

You can sideload version 18.04.02 which is still available and upgrade it bit a bit (I don't know how to do that) until 18.04.04.

See: https://wiki.ubuntu.com/WSL

### Installing Ubuntu on WSL by sideloading the .appx

https://aka.ms/wsl-ubuntu-1804

![image](https://user-images.githubusercontent.com/8397379/105839547-f4839600-5fd1-11eb-8b2d-b27a998bc77b.png)

Regards,
Carlos
